src/constants/constants.js

```diff
@@ -3,3 +3,57 @@ export const THEME_DARK = 'dark';
 
 export const CONTACT_US_PAGE_LINK = '/about';
 export const CITIES_PAGE_LINK = '/cities';
+
+export const API_AUTH_USERNAME = process.env.API_AUTH_USERNAME;
+export const API_AUTH_PASS = process.env.API_AUTH_PASS;
+export const API_BASE_URL = '/api'; //proxed to 'https://pfa.foreca.com' via webpack
+export const API_AUTH_URL = '/authorize/token';
+export const API_SEARCH_URL = '/api/v1/location/search/';
+export const API_GET_LOCATION_INFO_URL = '/api/v1/location/';
+export const API_CURRENT_WEATHER_URL = '/api/v1/current/';
+export const API_FORECAST_URL = '/api/v1/forecast/';
+export const API_FORECAST_3_HOURLY = '3hourly/';
```

Comment:
I have a gut feeling that you can pass any number here to get that much hourly forecast - is that so? Coz if so I'd parametrize this instead.